src/core/utils/EventManager.ts

```diff
@@ -14,15 +15,37 @@ export class EventManager {
         this.options = options;
         this.eventNormalizer = new EventNormalizer(editable, this._triggerEvent.bind(this));
     }
+
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+    /**
+     * Induce the user's intention from the received signal, based on the user's
+     * configuration and context.
+     * TODO: this is just a stub
+     *
+     * @param {Signal} signal
+     * @returns {Action}
+     */
+    _induceIntent(signal: Signal): Action {
```

Comment:
I'm not a fan of the word "induce", but I'm at a loss here, honestly.

Reply:
`toIntent` ? `signalToIntent` ? `matchIntent` ?